proofs/eo/cpc/programs/Nfa.eo

```diff
@@ -0,0 +1,155 @@
+; Declares a new sort which will be used to represent the states and transitions
+; of the automaton.
+(declare-type Nfa ())
```

Comment:
Can you add `@` as a prefix of all types/consts introduced? So that `@nfa.` is the prefix for each of those below and `@Nfa` is the type.